src/main/java/teammates/common/util/Const.java

```diff
@@ -639,6 +639,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_QUESTION_CONSTSUMNUMOPTION = "constSumNumOption";
         public static final String FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION = "constSumPointsPerOption";
```

Comment:
This particular constant can be quite confusing when compared to the newer ones; the constant here is used to determine if the points to be distributed is fixed, or a specified number multiplied by the number of options. As it is too risky to change this name right now, I'm leaving it as it is.